card/src/main/java/com/adyen/checkout/card/DefaultCardDelegate.kt

```diff
@@ -595,6 +596,17 @@ class DefaultCardDelegate(
         return cardOutputData.installmentOptions.isNotEmpty()
     }
 
+    override fun getCardListItems(): List<CardListItem> {
```

Comment:
Sorry I meant to do in it the output data not as a separate method. Also it seems that `supportedCardTypes` is only used for this adapter so maybe we can simply replace it with the mapped list? Does that work?

Reply:
This `CardListItem` class is just a representation of how we display the data, so I'm not sure if we should expose it to merchants in the output data 🤔 WDYT?

Reply:
Merchants can't see the output data anymore, it's strictly for the view now

Reply:
Oh nice! Then I'll put it in there